sdk-setup/src/customer-sign

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+set -e
+
+usage() {
+    echo "Simple wrapper over rpmsign --addsign"
+    echo "usage: $0 [-h] PACKAGE [PACKAGE ...]"
+    echo
+    echo "arguments:"
+    echo "  arguments for rpmsign --addsign"
+    echo "  -h, --help            show this help message and exit"
+}
+
+main() {
+    if [[ ( "$1" == "--help") ||  "$1" == "-h" ]]; then 
+        usage
+        exit 0
+    fi 
+
+    rpmsign --addsign "$@"
```

Comment:
If we need to support standard rpmsign, we need to obey its UI, so mb2 may not assume other options being accepted than those the standard rpmsing accepts. For that reason it makes more sense to hardcode use of '--addsign' option in mb2 and only allow to override the rpmsign executable. This could be done e.g. by looking for /usr/libexec/sdk-setup/sdk-rpmsign first and if that is not found continuing with rpmsign from PATH. I.e. no wrapper needs to be provided by default and if provided, mb2 will discover and use it instead of plain rpmsign.